### PR TITLE
Feature/merge 2.6.0 tag

### PR DESCRIPTION
**Pull request by thomasdanan**
merge tag

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## Release 2.6.0 (in development)
+## Release 2.6.1 (development in progress)
```
Comment (gdemonet) [inline CHANGELOG.md:3]: No, you want a new line for 2.6.1 above the line for 2.6.0 (and remove the `in development` part from the latter)

Comment (thomasdanan) [reply CHANGELOG.md:3]: okay

**Comment by NicolasT** [comment]
LGTM:

```
$ git fetch --prune
$ git branch --all --contains 2.6.0
  remotes/origin/feature/merge-2.6.0-tag
  remotes/origin/release/2.6.0
```

**Review by NicolasT — APPROVED** [review]

**Review by gdemonet — APPROVED** [review]

**Comment by thomasdanan** [comment]
coucou

**Pull request merged**